File: tests/human/x/dart/two-sum.dart
List<int> twoSum(List<int> nums, int target) {
  for (var i = 0; i < nums.length; i++) {
    for (var j = i + 1; j < nums.length; j++) {
      if (nums[i] + nums[j] == target) {
        return [i, j];
      }
    }
  }
  return [-1, -1];
}

void main() {
  var result = twoSum([2, 7, 11, 15], 9);
  print(result[0]);
  print(result[1]);
}

File: tests/human/x/kt/two-sum.kt
fun twoSum(nums: List<Int>, target: Int): List<Int> {
    val n = nums.size
    for (i in 0 until n) {
        for (j in i + 1 until n) {
            if (nums[i] + nums[j] == target) {
                return listOf(i, j)
            }
        }
    }
    return listOf(-1, -1)
}

fun main() {
    val result = twoSum(listOf(2, 7, 11, 15), 9)
    println(result[0])
    println(result[1])
}
